custom/icds_reports/views.py

```diff
@@ -2200,7 +2200,7 @@ def validate_param(self, state_id, month, year):
 
         is_valid = True
         error_message = ''
-        if date(2019, 12, 1) <= selected_month <= current_month:
+        if not (date(2019, 12, 1) <= selected_month <= current_month):
```

Comment:
trying to understand from the description,
was 
`date(2019, 12, 1) <= selected_month <= current_month`

should be 
`date(2019, 12, 1) > selected_month < current_month`?

Reply:
I think that wont work. Because this will give false in  conditions like where `selected_month=date(2022,1,1)` where it should give True and mark it as invalid.

Reply:
offlined.